src/store/store.js

```diff
@@ -0,0 +1,13 @@
+import { createStore, combineReducers } from 'redux';
+
+import { composeWithDevTools } from '@redux-devtools/extension';
+import userReducer from './user/reducer';
+
+const store = createStore(
+    combineReducers({
```

Comment:
why do we need combineReducers here? you are using one at the moment

Reply:
I made it just in case for the future, but right now it's not needed. Done

Reply:
No need as long as it's not used